test/ERC20RootVault.test.ts

```diff
@@ -0,0 +1,1149 @@
+                        const startNft = (
+                            await this.vaultRegistry.vaultsCount()
+                        ).toNumber();
+                        await ethers.provider.send("hardhat_setStorageAt", [
+                            this.vaultRegistry.address,
+                            "0x12", // address of vault
+                            "0x0000000000000000000000000000000000000000000000000000000000000000",
+                        ]);
+
+                        await withSigner(
+                            this.erc20VaultGovernance.address,
+                            async (signer) => {
+                                await expect(
+                                    this.subject
+                                        .connect(signer)
+                                        .initialize(
+                                            this.nft,
+                                            [
+                                                this.usdc.address,
+                                                this.weth.address,
+                                            ],
+                                            randomAddress(),
+                                            [startNft]
+                                        )
+                                ).to.be.revertedWith(
+                                    Exceptions.INVALID_INTERFACE // ADDRESS_ZERO
+                                );
+                            }
+                        );
+                    });
+                });
+
+                describe("when subvaultNFT does not support interface", () => {
+                    it(`reverts with ${Exceptions.INVALID_INTERFACE}`, async () => {
+                        const startNft = await this.vaultRegistry.vaultsCount();
+                        await withSigner(
+                            this.erc20VaultGovernance.address, // what to write here????
+                            async (signer) => {
+                                await expect(
+                                    this.subject
+                                        .connect(signer)
+                                        .initialize(
+                                            this.nft,
+                                            [
+                                                this.usdc.address,
+                                                this.weth.address,
+                                            ],
+                                            randomAddress(),
+                                            [startNft]
+                                        )
+                                ).to.be.revertedWith(
+                                    Exceptions.INVALID_INTERFACE
+                                );
+                            }
+                        );
+                    });
+                });
+
+                describe("when subvaultNFT supports interface", () => {
```

Comment:
WTF )))
even if you mean subVault instead of subVaultNft, if a subVault does NOT support the required interface, the transaction must be reverted with INVALID_INTERFACE